recipe/build.sh

```diff
@@ -29,6 +41,18 @@ for i in `ls`; do
         elif [[ $i == "include" ]]; then
             cp -rv $i ${PREFIX}/${targetsDir}
         elif [[ $i == "nvvm" ]]; then
+            for j in `find "${i}"`; do
+                if [[ "${j}" =~ /bin/.*$ ]]; then
+                    echo patchelf --force-rpath --set-rpath "\$ORIGIN/../lib64:\$ORIGIN/../../lib:\$ORIGIN/../../${targetsDir}/lib" "${j}" ...
+                    patchelf --force-rpath --set-rpath "\$ORIGIN/../lib64:\$ORIGIN/../../lib:\$ORIGIN/../../${targetsDir}/lib" "${j}"
+
+                # Do not patch libnvvm.so for now as it gets corrupted by patchelf. See build.sh for details
+                #elif [[ "${j}" =~ /lib.*/.*\.so($|\.) && ! -L "${j}" ]]; then
+                #    echo patchelf --force-rpath --set-rpath "\$ORIGIN" "${j}" ...
+                #    patchelf --force-rpath --set-rpath "\$ORIGIN" "${j}"
```

Comment:
This comment references `build.sh` for details however this is in `build.sh`. Are there more details we should add here or can that portion be dropped?

Reply:
Done. Ready for review.